README.md

```diff
@@ -1763,6 +1761,40 @@ A CQL querying example:
 http://localhost:<port>/configurations/entries?query=scope.institution_id=aaa%20sortBy%20enabled
 ```
 
+## Estimated totalRecords
+
+RMB adds a `totalRecords` field to result sets. It contains an estimation how many records were matching if paging parameters were set to `offset` = 0 and `limit` = unlimited.
+
+It uses this algorithm:
+
+1. Run "EXPLAIN SELECT" to get an estimation from PostgreSQL.
+2. If this is greater than 4*1000 return it and stop.
+3. Run "SELECT COUNT(*) FROM query LIMIT 1000".
+4. If this is less than 1000 return it (this is the exact number) and stop.
+5. If the result from 1. is less than 1000 return 1000 and stop.
```

Comment:
might be worth explicitly noting (this is where estimate < 1000 and actual >= 1000)

Reply:
Added totalRecords=1000 explanation.